inductiva/tasks/task.py

```diff
@@ -137,6 +137,19 @@ def get_status(self) -> models.TaskStatusCode:
 
         return models.TaskStatusCode(resp.body["status"])
 
+    def get_position_in_queue(self) -> Optional[int]:
+        """Get the position of the task in the queue.
+
+        This method issues a request to the API.
+        """
+        try:
+            resp = self._api.get_task_position_in_queue(self._get_path_params())
+            tasks_ahead = resp.body.get("tasks_ahead", None)
+            return tasks_ahead
+        except exceptions.ApiException as exc:
+            if exc.status == 404:
+                return None
```

Comment:
This should behave in the same manner as task.get_info() and task.info
task.get_position_in_queue() can do this and task.get_position_in_queue returns the value stored in memory.

Does this work in any kind of resource or only for tasks that run on the default queue ?

Reply:
I've talked with @sergiomsantos this is not live yet but the comment might still be relevant.

Reply:
> returns the value stored in memory

Okay! I will update the logic to store it in memory.

> Does this work in any kind of resource or only for tasks that run on the default queue ?

This work for every task that have a machine group, so I think it works for everything :)

Reply:
> @Pbarbosa92, please let me know if [a8e96d0](https://github.com/inductiva/inductiva/pull/1459/commits/a8e96d036937a127fd095ddaf6f1920836970ce8) is sufficient.

Now you can create 
```python
@property
def position_in_queue
```
That just returns the position in queue without any call to the api

Reply:
Hmmmm gotcha!
But since this isn't live for the other properties, should we wait for them to land first?

Reply:
Yes. We can refactor the task later. And make all the changes at the same time.